src/componentsEventPage/EventList.js

```diff
@@ -16,9 +16,9 @@ export default function EventList({eventArray, categoryFilter}) {
         <>
             {availableYears.map(year => (
                 <EventContainer key={year}>
-                    <EventYearHeadline>Event {year}</EventYearHeadline>
+                    <EventYearHeadline>Events {year}</EventYearHeadline>
                     {filteredEvents.map(event => event.eventStartDate.slice(0, 4) === year &&
-                        <EventItem event={event} key={event.id}></EventItem>
+                        <EventItem  event={event} key={event.id}></EventItem>
```

Comment:
Is the addition of an empty space in front of ```event={event}``` intentional?

Reply:
I think it makes no difference, but I will change it

Reply:
it was a copy past mistake